contrib/errorprone/README.md

```diff
@@ -0,0 +1,60 @@
+# Pants plugin for Error Prone
+
+The Error Prone plugin runs static analysis on Java source files and looks for various
+[bug patterns](http://errorprone.info/bugpatterns) not reported by the standard javac compiler.
+
+A full description of Error Prone can be found on the [Error Prone web page](http://errorprone.info/).
+
+The version of Error Prone used by the plugin requires Java 8 to run.  If you want to run Error Prone with Java 7 you will need to use version 2.0.5 or earlier. Using Java 7 may require changes to the bootclasspath to override certain classes from `rt.jar`.  See https://github.com/google/error-prone/issues/287 for more info.
```

Comment:
It might be good to note here how to change the version in config, or what you would need to change to do so.